### PR TITLE
JPF17 and memocode19 papers

### PR DESCRIPTION
**Pull request by Maryam81609**

### CODE REVIEW

File: _publications/2017_jpf_dr.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+authors:
+- dabaghchian
+- rakamaric
+
+journal-short: JPF
+year: 2017
+
+# bibentry: article | inproceedings | phdthesis | book
+bibentry: article
+bib:
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md:29]: You are missing a bunch of bibtex fields.
```
@article{jpf2017-dromt,
  author = {Maryam Dabaghchian and Zvonimir Rakamari\'c and
    Burcu K. Ozkan and Erdal Mutlu and Serdar Tasiran},
  title = {Consistency-Aware Scheduling for Weakly Consistent Programs},
  journal = {ACM SIGSOFT Software Engineering Notes},
  volume = {42},
  number = {4},
  year = {2017},
  month = {October},
  pages = {1--5},
  doi = {10.1145/3149485.3149493},
  publisher = {ACM},
  note = {Proceedings of the 2017 Java Pathfinder Workshop (JPF)},
}
```

File: _publications/2017_jpf_dr.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#  linksym: true
+
+# Link to the repository where the code is hosted
+code: https://github.com/Maryam81609/commander
+
+# Link to an official preprint server
+preprint_server: 
+#https://doi.org/10.1145/3149485.3149493
+
+abstract: "
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md]: Missing `<p>`

File: _publications/2017_jpf_dr.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Modern geo-replicated data stores provide high availability by relaxing the underlying consistency requirements. Programs layered over such data stores are called weakly consistent programs. Due to the reduced consistency requirements, they exhibit highly nondeterministic behaviors, some of which might violate program invariants. Therefore, implementing correct weakly consistent programs and reasoning about them is challenging. In this paper, we present a systematic scheduling approach that is aware of the underlying consistency model. Our approach dynamically explores all possible program behaviors allowed by the used data store consistency model, and it evaluates program invariants during the exploration. We implement the approach in a prototype model checker for Antidote, which is a causally consistent key-value data store with convergent conflict handling. We evaluate our tool on several benchmarks. The results show that our approach is effective in detecting buggy behaviors in weakly consistent programs.
+"
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md]: Missing `</p>`

File: _publications/2017_jpf_dr.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+# Use this if you have an external project website
+external-project: https://pages.lip6.fr/syncfree/index.html
+
+# The reference to the video entry
+video:
+# The reference to the preview viedo entry
+#preview-video:
+
+# the preprint
+pdf: 2017_jpf_dr.pdf
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md:59]: Use pdf from here: `/uusoc/res/soarlab/www-soarlab/html/publications`
The one you uploaded is messed up.

File: _publications/2017_jpf_dr.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+# A supplement PDF
+supplement: 2017_techrep_dr.pdf
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md:61]: Where is this file?

File: _publications/2017_jpf_dr.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+layout: publication
+# The quotes make the : possible, otherwise you can do it without quotes
+title: "Consistency-Aware Scheduling for Weakly Consistent Programs"
+# YYYY_CONFSHORTNAME_PAPERSHORTNAME
+key: 2017_jpf_dr
+
+# conference | journal | poster | workshop
+type: workshop
+redirect_from: /publications/2017_jpf_dr
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md]: This should be `/2017/09/jpf2017-dromt`

File: _publications/2019_memocode_dr.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+authors:
+- Dabaghchian
+- Rakamaric
+
+journal-short: MEMOCODE
+year: 2019
+
+#bibentry: article | inproceedings | phdthesis | book
+
+bibentry: inproceedings
```
Comment (zvonimir) [inline _publications/2019_memocode_dr.md:33]: You are missing a bunch of bibtex entries here.

File: _publications/2017_jpf_dr.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+# The shortname is used for auto-generated titles
+shortname: JPF 2017
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2017_jpf_dr.png
```
Comment (zvonimir) [inline _publications/2017_jpf_dr.md:15]: The actual file names have upper case PNG. So change that in file names of images.
Also, your teaser images are not both 1600x800. Please fix that too.

File: _publications/2019_memocode_dr.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2019_memocode_dr.png
+# 2018_tvcg_lineage.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2019_memocode_dr_teaser.dr
+#2018_tvcg_lineage_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- Dabaghchian
```
Comment (zvonimir) [inline _publications/2019_memocode_dr.md]: This should be lower-case.

**Review by zvonimir — CHANGES_REQUESTED** [review]

File: _publications/2017_jpf_dr.md (hunk 1 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md:29]: Done.

File: _publications/2017_jpf_dr.md (hunk 2 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md]: Done.

File: _publications/2017_jpf_dr.md (hunk 3 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md]: Done.

File: _publications/2017_jpf_dr.md (hunk 4 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md:59]: Done.

File: _publications/2017_jpf_dr.md (hunk 5 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md:61]: Done.

File: _publications/2017_jpf_dr.md (hunk 6 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md]: Done.

File: _publications/2019_memocode_dr.md (hunk 7 above)
Comment (Maryam81609) [reply _publications/2019_memocode_dr.md:33]: Done.

File: _publications/2017_jpf_dr.md (hunk 8 above)
Comment (Maryam81609) [reply _publications/2017_jpf_dr.md:15]: Done.

File: _publications/2019_memocode_dr.md (hunk 9 above)
Comment (Maryam81609) [reply _publications/2019_memocode_dr.md]: Done.

**Comment by Maryam81609** [comment]
I previewed my updates. I hope they are fine now.

**Review by zvonimir — APPROVED** [review]
Looks good to me.

**Pull request merged**